### DISCUSSION TITLE
SAML-based authentication?

### DISCUSSION THREAD

**Discussion by tyteen4a03** [keystonejs/keystone#7479] [Questions] [answered]
Is it possible to use SAML-based authentication instead of the built-in authentication solution?

**Comment by raveling** [comment, marked as answer]
@tyteen4a03 It's possible but there's no example of doing it. @borisno2 has an [OIDC plugin](https://github.com/OpenSaasAU/keystone-nextjs-auth) that uses Next Auth if you'd like to use that.
